awscli/paramfile.py

```diff
@@ -37,6 +37,10 @@
     'cloudformation.update-stack.stack-policy-url',
     'cloudformation.set-stack-policy.stack-policy-url',
 
+    'custom.cp.website-redirect',
```

Comment:
On a side note, I do not like that all custom commands have ``custom`` as its service in the event that is emitted. It is not good if there are ever custom commands that overlap in the cli namespace even though they may have a different command lineage.

However, I do not think any change can be made till after the command lineage PR is merged.